Fastlane/deployment_lanes.rb

```diff
@@ -325,6 +325,12 @@
   UI.message "IPA saved at #{ENV['IPA_OUTPUT_PATH']}"
 end
 
+desc 'Just install the certs onto bitrise so this can build for test lab'
```

Comment:
```suggestion
desc 'install the certificates onto Bitrise so this can build for test lab'
```

Can we also add some context as to what "test lab" is referring to?